packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -120,6 +120,13 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       }),
     },
 
+    headState: {
+      unfinalizedPubkeyCacheSize: register.gauge({
+        name: "head_state_unfinalized_pubkey_cache_size",
```

Comment:
prefix in metric name is missing

```suggestion
        name: "beacon_head_state_unfinalized_pubkey_cache_size",
```

I also noticed all of the metrics added in this branch are not used anywhere